### ISSUE TITLE
State-of-the-art comparison

### ISSUE THREAD

**Issue by xl-sr** [microsoft/VQ-Diffusion#4]
Table from the recent "Improved Vector Quantized Diffusion Models" comparing to "state-of-the-art".

![image](https://user-images.githubusercontent.com/29833625/171329548-d25ad486-2f02-41f7-8446-c4d32d58f24f.png)

It seems that you're not aware of all previous approaches (disregarding concurrent work like Make-a-scene and Imagen):

**COCO** 
As I understand, your approach has been trained on COCO, hence, it is not fair to compare it to GLIDE directly (which is zero-shot). In the training+testing on coco setting, prior work also achieved better numbers (FID=8.12).
- https://paperswithcode.com/paper/lafite-towards-language-free-training-for

**ImageNet**
Current best FID=2.26 at resolution 256x256.
- https://paperswithcode.com/sota/image-generation-on-imagenet-256x256

**Comment by cientgu** [comment]
Thanks for your question. Since other zero-shot methods including GLIDE has ten times more parameters than our model, it's hard to do a fair comparison.

**Comment by xl-sr** [comment]
Fair point, but then why include GLIDE at all, one difference does not even out the other. Currently, it's comparing apples and oranges. Instead, the comparison should include other approaches like Lafite and Make-a-scene, both performing better.

Also, the point with Imagenet still stands. Sorry to be so adamant, but I believe numbers of prior literature need to be faithfully reported to actually assess progress.
